### DISCUSSION TITLE
phantom-procs aren't killed on One UI 4.1

### DISCUSSION THREAD

**Discussion by Rudxain** [termux/termux-app#3480] [Q&A] [answered]
I have a SM-A315G with Android **12.0**, yet I haven't experienced any of the problems described in #2366 . Did Samsung really backport the fix from v13/12L/12.1 into 12.0? Or, are my usage-habits biasing my results?

I regularly use (and keep running) programs such as `bc`, `dash`, `adb` server, etc... I even switch to other apps, then switch back to Termux (using Recent-Apps), minutes (and sometimes hours) of idle time, and the system never kills any process.

I even **disabled "RAM Plus"**, and LMK is just chilling as if everything is ok.

No, I didn't follow [these instructions](https://github.com/termux/termux-app/issues/2366#issuecomment-1237468220).

Ironically, despite [Samsung's reputation](https://dontkillmyapp.com/samsung), everything works fine.

What's going on?

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/agnostic-apollo/Android-Docs/blob/master/en/docs/apps/processes/phantom-cached-and-empty-processes.md#how-to-manually-trigger-phantom-process-killing-in-termux

**Comment by Rudxain** [comment]
@agnostic-apollo I ran the cmd, and (sadly) it worked immediately!

So it does seem my habits have "saved" me. I thought the PPK was more aggressive
